### PR TITLE
design v1 - navbar, login/signup forms, success pages, error modal

### PR DESCRIPTION
**Pull request by e91096**
[Trello](https://trello.com/c/RS5zy1lJ/40-design-10)

✨ styles:
- navbar
- login form
- signup form
- login success page
- registration success page
- form error modal

📱  also adds responsiveness to above pages 😱 

🚫 does not style:
- campaign index
- campaign edit
- bill summary index
- bill summary edit

❓ thoughts/suggestions?
- i'm looking into replacing `Menu` with hamburger/kabob menus but open to anything else!!

## Notes for Reviewer
- everything should still work (inputs, submission, navigating via the navbar, etc)
- styles should feel cohesive, experience should feel intuitive
- styles should look reasonable(/pretty damn good) on a mobile device

### take 1:
screen shot 2018-09-16 at 9 59 11 pm

#### login
screen shot 2018-09-23 at 12 21 38 am

#### drop down
screen shot 2018-09-23 at 1 39 33 am

#### errors
screen shot 2018-09-23 at 12 28 38 am

#### success
screen shot 2018-09-23 at 1 37 55 am

#### mobile!
screen shot 2018-09-23 at 1 24 41 am

### CODE REVIEW

File: frontend/stylesheets/input.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  padding: 20px 0;
+}
+
+.input-container input {
+  font-size: 18px;
+  margin: 10px 0;
+  padding: 10px;
+}
+
+@media only screen and (max-width: 700px) {
```
Comment (brycemcd) [inline frontend/stylesheets/input.css]: WHA?!? a media query?!? Our mostly mobile users are getting some love! :medal_sports: hurray!

**Comment by brycemcd** [comment]
@e91096 this looks really nice and creates a good scaffold for future pages.

I notice that on login/signup, the navigation allows for getting to campaigns and bill summaries. Can you abstract the menu into something like "admin" (editor) vs. "constituent" ? We don't want to send constituents to pages where they can edit campaigns. (I know that we have ACLs to protect this though too.)

**Comment by e91096** [comment]
yeah absolutely! just having the signup/login/logout options is good?

**Comment by brycemcd** [comment]
Yeah, those links are good for constituents. We'll want to add something for invitations soon too

File: frontend/components/Navbar.jsx [hunk 2]
```diff
@@ -1,32 +1,112 @@
...
+              key={item.name}
+            />);
+        });
+      }
+
+      navItems.push(
+        <NavItem
+          name="Log out"
+          path="/"
+          handleClick={() => { this.props.logoutUser(() => push('/login'), _.noop); }}
```
Comment (brycemcd) [inline frontend/components/Navbar.jsx]: Why is the `_.noop` needed?

File: frontend/stylesheets/login_form.css [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    word-wrap: break-word;
+  }
+
+  div.form-container h1 {
+    font-size: 42px;
+    font-weight: 700;
+  }
+
+  div.form-container h2 {
+    font-size: 30px;
```
Comment (brycemcd) [inline frontend/stylesheets/login_form.css]: Is pixels the right thing to use here instead of ems?

File: frontend/stylesheets/nav_bar.css [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+div.logo-icon > span:last-child {
+  color: #A0D2CF;
+}
+
+nav.nav-container > ul.nav-menu {
+  align-self: flex-end;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  z-index: 1;
```
Comment (brycemcd) [inline frontend/stylesheets/nav_bar.css]: Thank you for making this 1 and not 10000 or some strange huge number. I want to punch kittens when I see that.

File: frontend/stylesheets/login_form.css (hunk 3 above)
Comment (e91096) [reply frontend/stylesheets/login_form.css]: oh! i've just always used px for really no good reason. i can do ems!

**Pull request merged**